server/views/pages/activities/activitiesMigration.njk

```diff
@@ -110,6 +122,13 @@
                             </div>
                           {% endif %}
                         </div>
+
+                        <div class="govuk-grid-row govuk-!-margin-bottom-2">
+                          <div class="govuk-grid-column-one-third">
+                            <a data-qa="migrate-allocations-link" class="govuk-link" href="/allocations-migration/start?prisonId={{ u.filterPrisonId }}&courseActivityId={{ u.filterCourseActivityId }}">Migrate Allocations</a>
```

Comment:
Add a link to the Allocations migration above the "end activities" button as this must be done first.